package.json

```
@@ -17,7 +17,7 @@
     "test": "jest",
     "preversion": "echo $npm_package_version > .old-version.txt",
     "version": "sed -i '' \"s/$(cat .old-version.txt)/$npm_package_version/\" README.md && git add README.md",
-    "postversion": "rm -rf .old-version.txt"
+    "postversion": "rm -rf .old-version.txt && ./bin/set-major-version-tag.js"
```

to avoid needing executable permissions, this should be fine:

```suggestion
    "postversion": "rm -rf .old-version.txt && node ./bin/set-major-version-tag.js"
```

since it already works, this should be fine